tools/block-generator/Makefile

```diff
@@ -22,15 +23,29 @@ cleanup: clean-reports
 enter-pg:
 	docker exec -it generator-test-container psql -U algorand -d generator_db
 
-run-runner:
+clean:
+	rm -f block-generator
+
+block-generator: clean
+	go build
```

Comment:
Consider moving this to the first target so that `make` with no arguments builds the binary.

Reply:
53c4960b9503999dd64dfaa9ffaceac9ffefc6d9